requirements.txt

```diff
@@ -1,157 +1,157 @@
-aniso8601==8.0.0
-anyio==2.2.0
-appnope==0.1.2
-arcgis==1.8.5.post3
-argon2-cffi==20.1.0
-arviz==0.11.2
-async-generator==1.10
-attrs==19.3.0
-Babel==2.9.0
-backcall==0.2.0
-bleach==3.3.0
-boto3==1.16.63
-botocore==1.19.63
-cachetools==4.2.1
-certifi==2020.4.5.1
-cffi==1.14.5
-cftime==1.4.1
-chardet==3.0.4
-click==7.1.1
-click-plugins==1.1.1
-cligj==0.7.1
-cryptography==3.4.7
-cycler==0.10.0
-Cython==0.29.22
-decorator==4.4.2
-defusedxml==0.7.1
-deprecation==2.1.0
-entrypoints==0.3
-factory-boy==3.1.0
-Faker==4.14.0
-Fiona==1.8.19
-Flask==1.1.2
-Flask-Cors==3.0.9
-Flask-Migrate==2.5.3
-flask-restplus==0.13.0
-Flask-Script==2.0.6
-Flask-SQLAlchemy==2.4.4
-future==0.18.2
-google-api-core==1.26.0
-google-api-python-client==1.12.8
-google-auth==1.27.0
-google-auth-httplib2==0.0.4
-google-auth-oauthlib==0.4.2
-googleapis-common-protos==1.53.0
-gunicorn==20.0.4
-httplib2==0.19.0
-idna==2.9
-importlib-metadata==4.0.1
-iniconfig==1.1.1
-ipykernel==5.4.3
-ipython==7.20.0
-ipython-genutils==0.2.0
-ipywidgets==7.6.3
-itsdangerous==1.1.0
-jedi==0.18.0
-Jinja2==2.11.2
-jmespath==0.10.0
-json5==0.9.5
-jsonschema==3.2.0
-jupyter-client==6.1.11
-jupyter-core==4.7.1
-jupyter-packaging==0.9.2
-jupyter-server==1.6.4
-jupyterlab==3.0.14
-jupyterlab-pygments==0.1.2
-jupyterlab-server==2.5.0
-jupyterlab-widgets==1.0.0
-keyring==23.0.1
-kiwisolver==1.3.1
-lerc==0.1.0
-Mako==1.1.3
-MarkupSafe==1.1.1
-marshmallow==3.8.0
-matplotlib==3.3.4
-mistune==0.8.4
-more-itertools==8.4.0
-munch==2.5.0
-nbclassic==0.2.7
-nbclient==0.5.3
-nbconvert==6.0.7
-nbformat==5.1.3
-nest-asyncio==1.5.1
-netCDF4==1.5.6
-notebook==6.3.0
-ntlm-auth==1.5.0
-numpy==1.19.2
-oauthlib==3.1.0
-packaging==20.4
-pandas==1.0.3
-pandocfilters==1.4.3
-parso==0.8.1
-patsy==0.5.1
-pexpect==4.8.0
-pickleshare==0.7.5
-Pillow==8.1.0
-pluggy==0.13.1
-print-dict==0.1.14
-prometheus-client==0.10.1
-prompt-toolkit==3.0.14
-protobuf==3.15.3
-psycopg2-binary==2.8.5
-ptyprocess==0.7.0
-py==1.9.0
-pyairtable==1.0.0.post1
-pyasn1==0.4.8
-pyasn1-modules==0.2.8
-pyclipper==1.2.1
-pycparser==2.20
-Pygments==2.7.4
-pyparsing==2.4.7
-pyproj==3.0.1
-pyrsistent==0.16.0
-pyshp==2.1.3
-pystan==2.19.1.1
-pytest==6.1.1
-python-certifi-win32==1.6
-python-dateutil==2.8.1
-python-dotenv==0.12.0
-python-editor==1.0.4
-python-json-logger==2.0.1
-pytz==2019.3
-pyzmq==22.0.2
-requests==2.24.0
-requests-ntlm==1.1.0
-requests-oauthlib==1.3.0
-requests-toolbelt==0.9.1
-rsa==4.7.2
-s3transfer==0.3.4
-scipy==1.5.0
-Send2Trash==1.5.0
-setuptools-scm==6.0.1
-Shapely==1.7.1
-six==1.14.0
-slack-sdk==3.2.0
-sniffio==1.2.0
-SQLAlchemy==1.3.18
-statsmodels==0.12.2
-terminado==0.9.4
-testpath==0.4.4
-text-unidecode==1.3
-toml==0.10.1
-tomlkit==0.7.0
-tornado==6.1
-traitlets==5.0.5
-typing-extensions==3.7.4.3
-ujson==4.0.2
-uritemplate==3.0.1
-urllib3==1.25.9
-wcwidth==0.2.5
-webencodings==0.5.1
-Werkzeug==0.16.0
-widgetsnbextension==3.5.1
-wrapt==1.12.1
-xarray==0.17.0
-yapf==0.30.0
-zipp==3.4.1
\ No newline at end of file
+alembic~=1.4.2
```

Comment:
This is fine. can you just add a comment at the top of this file explaining what ~= will do?

Reply:
Done.